src/js/containers/dashboard/ActiveDashboardImpactsContainer.jsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * ActiveDashboardImpactsContainer.jsx
+ * Created by Daniel 4/1/2020
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+import { connect } from 'react-redux';
+
+import * as DashboardHelper from 'helpers/dashboardHelper';
+import ActiveDashboardImpacts from 'components/dashboard/impacts/ActiveDashboardImpacts';
+
+const propTypes = {
+    appliedFilters: PropTypes.object.isRequired,
+    errorLevel: PropTypes.string,
+    submissionID: PropTypes.string
+};
+
+export class ActiveDashboardImpactsContainer extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            submissionData: {},
+            inFlight: false,
+            hasFailed: false
+        };
+
+        this.getImpacts = this.getImpacts.bind(this);
+    }
+
+    componentDidMount() {
+        this.getImpacts();
+    }
+
+    componentDidUpdate(prevProps) {
+        if (!_.isEqual(prevProps, this.props)) {
+            this.getImpacts();
+        }
+    }
+
+    getImpacts() {
+        this.setState({
+            inFlight: true
+        });
+        const submissionId = parseInt(this.props.submissionID, 10);
+        const fileType = this.props.appliedFilters.file;
+        const errorLevel = this.props.errorLevel;
+        DashboardHelper.fetchActiveImpacts(submissionId, fileType, errorLevel)
+            .then((data) => {
+                this.setState({
+                    hasFailed: false,
+                    submissionData: data,
+                    inFlight: false
+                });
+            })
+            .catch((err) => {
+                console.error(err);
+                this.setState({
+                    hasFailed: true,
```

Comment:
Looks like we have [DEV-4320](https://federal-spending-transparency.atlassian.net/browse/DEV-4320) to ensure error/loading/no results states, but we should probably check with @AGuyNamedMarco ahead of time to see what he thinks that would look like for this section.

Reply:
Including JIRA ticket to address this: https://federal-spending-transparency.atlassian.net/browse/DEV-4811

Reply:
Thanks!